src/main/java/com/marginallyclever/makelangelo/preview/PreviewPanel.java

```diff
@@ -152,10 +174,12 @@ public void reshape(GLAutoDrawable glautodrawable, int x, int y, int width, int
 
 		gl2.glMatrixMode(GL2.GL_PROJECTION);
 		gl2.glLoadIdentity();
+		glu.gluOrtho2D(-width/2, width/2, -height/2, height/2);
+		/*
 		glu.gluPerspective( 90,
```

Comment:
please remove commented code

Reply:
Personally I would have a hard time remember how to switch back to perspective mode, this comment may be useful in future.

Reply:
git is the memory of the code that is why you have to create some meaningful commits messages in order to find old code. Comments are used to explain why how the code works if it is uncleared.
In intellij, right click on your file > Git > Show history and you'll find all the modifications done on this file.